tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -226,7 +226,7 @@ def montecarlo_main_loop(
 
         vpacket_collection = vpacket_collections[i]
         rpacket_tracker = rpacket_trackers[i]
-
```

Comment:
Can you please remove this change from the commit history?

Reply:
Sure.